iron-focusables-helper.html

```diff
@@ -127,18 +127,17 @@
         //  </div>
         // TODO(valdrin) support ShadowDOM v1 when upgrading to Polymer v2.0.
         var children;
-        if (element.localName === 'content') {
+        if (element.localName === 'content' || element.localName === 'slot') {
```

Comment:
Note to self:
taborder w/o ShadowDOM: http://output.jsbin.com/rotohan/3/
taborder in SD v0: http://output.jsbin.com/tavedi/3/
taborder in SD v1: http://output.jsbin.com/vaciwaw/3/